README.md

```diff
@@ -74,6 +74,17 @@ webex.rooms.create({ title: `My First Room` }).then(room => {
 
 #### _A note on browser usage_
 
+We do provide a built, minified version of the SDK, that includes `window.webex`, which is hosted on our repo and can be used with either [unpkg](https://unpkg.com/), [jsdelivr](https://jsdelivr.com/), or [gitcdn.xyz](https://gitcdn.xyz/).
```

Comment:
```suggestion
We do provide a built, minified version of the SDK, that includes `window.Webex`, which is hosted on our repo and can be used with either [unpkg](https://unpkg.com/), [jsdelivr](https://jsdelivr.com/), or [gitcdn.xyz](https://gitcdn.xyz/).
```

Since we export `Webex` https://github.com/webex/webex-js-sdk/pull/1573/files#diff-11e9f7f953edc64ba14b0cc350ae7b9dR20